examples/pysdl2.py

```diff
@@ -0,0 +1,357 @@
+"""
+ Simple SDL2 / cefpython3 example.
+
+ Only handles mouse events but could be extended to handle others.
```

Comment:
This needs to be updated. It should say that keyboard and mouse support is incomplete:
- Keyboard support may be extended to support:
  - marking text in inputs with the shift key
  - selecting all text with ctrl+a
  - not all special keys may be supported, but can be extended by modifying the getKeyCode() function
- Mouse support may be extended to support:
  - mouse hover by calling SendMouseMoveEvent()
  - mouse wheel scrolling by calling SendMouseWheelEvent()